### PR TITLE
Add ability to get specific download by id

### PR DESCRIPTION
**Pull request by gtimti**
This pull request adds ability to retrieve data for specific download.

https://www.pivotaltracker.com/story/show/122045997

### CODE REVIEW

File: lib/gap_intelligence/client/requestable.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'uri'
```
Comment (shavkat-samatov) [inline lib/gap_intelligence/client/requestable.rb]: do we need to require uri here? I see we require it in `lib/gap_intelligence.rb'

File: lib/gap_intelligence/models/download.rb [hunk 2]
```diff
@@ -3,11 +3,12 @@ class Download < Record
     attributes :start_at, :end_at, class: Date
     attribute :created_at, class: Time
 
-    attributes :category_names
-    attribute :country_names, class: Array
-    attribute :report_types, class: Array
+    attributes :categories, class: Array
+    attribute :category_names
+    attributes :countries, :country_names
```
Comment (shavkat-samatov) [inline lib/gap_intelligence/models/download.rb]: should countries and country_names be Array as well?

**Comment by shavkat-samatov** [comment]
minor comments, everything else looks good to me! 👍

**Comment by gtimti** [comment]
@shavkat-samatov , thank you for review, I removed extra require uri and made category_names to be an array.

**Comment by shavkat-samatov** [comment]
👍 looks great to me!

**Pull request merged**